habitat/utils/gym_adapter.py

```diff
@@ -152,29 +227,34 @@ def __init__(self, env, save_orig_obs: bool = False):
 
         self._env = env
 
-    def step(self, action: np.ndarray):
-        action_args = {}
-        for k, (start_i, end_i) in self.action_mapping.items():
-            action_args[k] = action[start_i:end_i]
-        action = {
-            "action": self.orig_action_name,
-            "action_args": action_args,
-        }
-        return self.direct_hab_step(action)
+    def step(self, action: Union[np.ndarray, int]):
+        assert self.action_space.contains(
```

Comment:
This is common practice for gym environments. I know Andrew wants to eventually be able to pass an ID as action. I think we should eventually give an option to set arbitrary low and high bounds to the action space (not in this PR)